app/earn/[...vault]/header/first-line/vault-title.tsx

```diff
@@ -1,30 +1,51 @@
 import type { PropsWithChildren } from 'react'
-import { ExternalAction } from '../second-line/actions'
+import { useVaultContext } from '../../hooks/use-vault-context'
 import styles from './vault-title.module.scss'
 
 import { Tag } from '@/components/chip/tag'
+import { getChainExplorer } from '@/providers/wallet/wrappers/helpers'
+import { WrapperLink } from '@/components/links/wrapper-link'
+import IconExternal from '@/components/icons/icon-external'
 
 export function VaultTitle({ children }: PropsWithChildren) {
+  return <div className={styles.container}>{children}</div>
+}
+
+function HowVaultWorks() {
   return (
-    <div className={styles.container}>
-      {children}
-    </div>
+    <WrapperLink href="https://docs.eonian.finance/basics/how-eonian-works" icon={<IconExternal />} iconAtEnd>
+      How Vault works?
+    </WrapperLink>
```

Comment:
It should handle it automatically
```suggestion
    <WrapperLink href="https://docs.eonian.finance/basics/how-eonian-works" withIcon>
      How Vault works?
    </WrapperLink>
```

Reply:
it doesn't for some reason 🤔